app/helpers/Emails/NotificationsHelper/Comments/assignmentSolutionComment.latte

```diff
@@ -0,0 +1,6 @@
+<h3>New comment at assignment '{$assignment}' submitted by '{$solutionAuthor}'</h3>
+<p>
+    New comment was added under the solution of assignment '{$assignment}'
+    submitted by '{$solutionAuthor}'. Comment was added by '{$author}' on
+    {$date|date:'j.n.Y H:i'}.
+</p>
```

Comment:
I think it would be very helpful if the text of the comment was included as well. In fact, the text is far more important then the exact date-time! It would also be swell, if URL pointing to the solution (where the conversation is) was included.

Reply:
Do we already have a config option to specify the address of the web app?

Reply:
well we need the route for assignment, which ideally should be also configurable...